RevenueCat/Plugins/Android/PurchasesWrapper.java

```diff
@@ -386,8 +403,7 @@ private static void logJSONException(JSONException e) {
         Log.e("Purchases", "JSON Error: " + e.getLocalizedMessage());
     }
 
-    private static void sendJSONObject(JSONObject object, String method) {
-        Log.e("Purchases", object.toString());
```

Comment:
removed these logs because they were making the logcat super poluted

Reply:
Right, additionally, we should be careful with all these hybrid side logs. As I mentioned before, ideally they would be rerouted to the same log handler as well so devs can handle all logs.

I think this can be done in a different PR but we definitely shouldn't lose track of it.